plugins/inputs/system/ps.go

```diff
@@ -99,21 +101,26 @@ func (s *systemPS) DiskUsage(
 			}
 		}
 
+		// If there's a host mount prefix, exclude other paths
+		if len(hostMountPrefix) > 0 && !strings.HasPrefix(p.Mountpoint, hostMountPrefix) {
```

Comment:
This should a separate behavior IMO.
The way I understand the `HOST_MOUNT_PREFIX`option is to be able to remove this prefix from the pahts in the mountpoint stats, not specifically to filter those only.
Maybe this should be added has a separate option. This would be more explicit.